setup.py

```diff
@@ -56,6 +56,7 @@ def read_file(fname):
 install_requires = [
     'colorlog',
     'django>=1.9,<1.10',
+    'django-filter==0.13.0',
```

Comment:
Can you make this a greater than comparison? Just so we don't have to keep changing this in the future.